avae/data.py

```diff
@@ -188,6 +194,9 @@ def __init__(
         amatrix=None,
         classes=None,
         transform=None,
+        gaussian_blur=False,
+        normalise=False,
+        shift_min=False,
```

Comment:
Add to shift-mean to docstring.

Reply:
fixed in [f1c9aca](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/f1c9acad22d140675956401d316dbd196aea0663)